mlrun/config.py

```diff
@@ -303,6 +303,7 @@
         },
         "v3io_api": "",
         "v3io_framesd": "",
+        "default_service_account": "",
```

Comment:
We introduced new config attribute which is called `function` that inside of it we define `spec` and inside of it the function spec attributes, let's move it to there.